docs/usage/usage_advanced_debug.rst

```diff
@@ -64,11 +64,11 @@ not be sent the message. Putting a try/except clause around every sendMessage is
 typically not practical. 
 
 Since exceptions are common during application development (bugs due to 
-invalid arguments, failed assertions, etc), Pypubsub provdes a hook to register
+invalid arguments, failed assertions, etc...), Pypubsub provides a hook to register
```

Comment:
I would argue there should just be a single period here after the etc.  An ellipsis (...) is somewhat redundant with etc.